### ISSUE TITLE
load libtorch2trt_dynamic.so error

### ISSUE THREAD

**Issue by Chen-cyw** [grimoire/torch2trt_dynamic#11]
I export the libtorch2trt_dynamic.so by "sudo python setup.py develop --plugins" and configure the trt and cuda path, but when I use the  .so , it always get the error
 "OSError: libtorch2trt_dynamic.so: undefined symbol: _ZNK6google8protobuf7Message11GetTypeNameEv"
did you try and finish the whole build&load process

**Comment by grimoire** [comment]
`--plugins` flag is not used in this repo. Please install follow  the README of this repo, not the official one.

**Comment by Chen-cyw** [comment]
> `--plugins` flag is not used in this repo. Please install follow the README of this repo, not the official one.

I config the setup.py like this:
def initialize_plugins_options(cmd_obj):
    cmd_obj.plugins = True
    cmd_obj.cuda_dir = '/home/cyw/cuda-10.2'
    cmd_obj.torch_dir = None
    cmd_obj.trt_inc_dir = '/home/cyw/TensorRT-7.2.1.6/targets/x86_64-linux-gnu/include'
    cmd_obj.trt_lib_dir = '/home/cyw/TensorRT-7.2.1.6/targets/x86_64-linux-gnu/lib'
any other things I need to do?

**Comment by grimoire** [comment]
Please follow [these](https://github.com/grimoire/torch2trt_dynamic#setup) steps.
you do **NOT** need to change anything inside setup.py, The plugin used in this repo is [amirstan_plugin](https://github.com/grimoire/amirstan_plugin).

**Comment by Chen-cyw** [comment]
> Please follow [these](https://github.com/grimoire/torch2trt_dynamic#setup) steps.
> you do **NOT** need to change anything inside setup.py, The plugin used in this repo is [amirstan_plugin](https://github.com/grimoire/amirstan_plugin).
thanks!
I want to use the converted model by mmdetection-to-tensorrt in pure tensorrt env(later in  tensorrt server),   Is that  I just need to load the "libamirstan_plugin.so"  in tensortrt ?  the torch2trt_dynamic  is only for  convert ,not for runtime?

**Comment by grimoire** [comment]
Yes, read [this](https://github.com/grimoire/mmdetection-to-tensorrt/blob/master/docs/getting_started.md#use-in-c) for detail.
